README.md

```diff
@@ -8,16 +51,222 @@ so don't try to use it for anything close to a production environment.
 
 ***Note: this README is only for development purposes.***
 
-## Quick Start
 
-### Build binary
+## <a name='OperatorDeployment'></a>Operator Deployment
+
+The ORAN O2 IMS implementation in OpenShift is managed by the IMS operator. 
+It configures the different components defined in the specification: 
+the deployment manager service, the resource server, alarm server, subscriptions to resource and alert.
+
+The IMS operator will create an O-Cloud API that will be available to be queried, for instance from a SMO. It
+also provides a configuration mechanism using a Kubernetes custom resource definition (CRD) that allows the hub cluster administrator to configure the different IMS microservices properly.
+
+
+### <a name='Deploytheoperatoronyourcluster'></a>Deploy the operator on your cluster
+
+The IMS operator is installed on an OpenShift cluster where Red Hat Advanced Cluster Management for Kubernetes (RHACM), a.k.a. hub cluster, is installed too. Let’s install the operator. You can use the latest automatic build from the [openshift-kni namespace](https://quay.io/repository/openshift-kni/oran-o2ims-operator) in quay.io or build your container image with the latest code. 
+
+If you want to build the image yourself and push it to your registry right after:
+
+> :warning: Replace the IMAGE_NAME value with the full name of your container image
+
+```bash
+$ export IMAGE_NAME=quay.io/alosadag/oran-o2ims:latest
+$ git clone https://github.com/openshift-kni/oran-o2ims.git
+$ cd oran-o2ims
+$ make docker-build docker-push CONTAINER_TOOL=podman IMG=${IMAGE_NAME}
+
+..REDACTED..
+Update dependencies
+hack/update_deps.sh
+hack/install_test_deps.sh
+Downloading golangci-lint
+…
+[3/3] STEP 5/5: ENTRYPOINT ["/usr/bin/oran-o2ims"]
+[3/3] COMMIT quay.io/alosadag/oran-o2ims:v4.16
+--> eaa55268bfff
+Successfully tagged quay.io/alosadag/oran-o2ims:latest
+eaa55268bfffeb23644c545b3d0a768326821e0afea8b146c51835b3f90a9d0c
+```
+
+Now, let's deploy the operator. If you want to deploy your already built image then add the `IMG=${IMAGE_NAME}` argument to the `make` command:
+
+```sh
+$ make deploy install
+
+… REDACTED …
+Update dependencies
+hack/update_deps.sh
+hack/install_test_deps.sh
+Downloading golangci-lint
+… REDACTED …
+/home/alosadag/Documents/CNF/ORAN/oran-o2ims/bin/kustomize build config/default | /home/alosadag/Documents/CNF/ORAN/oran-o2ims/bin/kubectl apply -f -
```

Comment:
Should probably remove the path and just start with .../oran-o2ims - also should remove any occurrences of your username and replace with <user> (maybe those will all disappear with the path though).